### PR TITLE
Simple post

### PR DESCRIPTION
**Pull request by Pawel-Szydlo**

### CODE REVIEW

File: src/components/Blog/SinglePost/style.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    fontSize: '22px',
+    lineHeight: '40px',
+    fontWeight: 700,
+    padding: '8px 16px 8px 16px',
+    borderRadius: '10px',
+    marginTop: '50px',
+    marginBottom: '20px',
+    width: 'max-content'
+  }
+  // title: {
```
Comment (awojciak) [inline src/components/Blog/SinglePost/style.ts:87]: maybe remove this commented code

File: src/components/Blog/SinglePost/style.ts [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+const useStyles = makeStyles((theme: Theme) => ({
+  mainContainer: {
+    fontFamily: 'Inter',
+    background: colors.navy.background,
+    display: 'flex',
+    flexDirection: 'column',
+    alignContent: 'center',
+    alignItems: 'center',
+    color: '#FFFFFF',
```
Comment (awojciak) [inline src/components/Blog/SinglePost/style.ts:12]: colors.white.main

File: src/components/Blog/SinglePost/SinglePost.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Button, Grid } from '@material-ui/core'
+
+import ReactMarkdown from 'react-markdown'
+import remarkGfm from 'remark-gfm'
+import useStyles from './style'
+const id: number = 1
+const post: string = `
```
Comment (awojciak) [inline src/components/Blog/SinglePost/SinglePost.tsx:8]: move mock data to story

File: src/components/Blog/SinglePost/SinglePost.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+`
+
+export const SinglePost: React.FC = () => {
+  const classes = useStyles()
+
+  return (
+    <Grid key={ id } container item justifyContent='center' className={classes.mainContainer}>
+      <ReactMarkdown children={ post } remarkPlugins={[remarkGfm]} />
+      <Button className={classes.backButton}>Back to Blog</Button>
```
Comment (awojciak) [inline src/components/Blog/SinglePost/SinglePost.tsx:46]: use translation mechanism

**Review by awojciak — CHANGES_REQUESTED** [review]
obraz
why is the post text centered and text in button uppercased?

**Review by Kimi82 — APPROVED** [review]

**Review by awojciak — APPROVED** [review]

**Pull request merged**